client/allocrunner/groupservice_hook_test.go

```diff
@@ -0,0 +1,79 @@
+package allocrunner
+
+import (
+	"testing"
+
+	"github.com/hashicorp/nomad/client/allocrunner/interfaces"
+	"github.com/hashicorp/nomad/client/consul"
+	"github.com/hashicorp/nomad/helper/testlog"
+	"github.com/hashicorp/nomad/nomad/mock"
+	"github.com/hashicorp/nomad/nomad/structs"
+	"github.com/stretchr/testify/require"
+)
+
+var _ interfaces.RunnerPrerunHook = (*groupServiceHook)(nil)
```

Comment:
Is this just asserting the interface compiles?

Reply:
Yeah, we do it in all of our `*_hook_test.go` files to ensure we didn't typo something. A little gross but handy.